pyClarion/base/components.py

```diff
@@ -219,12 +219,19 @@ class Assets(SimpleNamespace): # type: ignore
 
 
 class Domain(object):
-    """A feature domain."""
+    """A feature domain. It is used for definition of elements relevant to the 
+    simulation. Domain can be viewed as a collection of element.
```

Comment:
'Element' is too generic, `Domain` objects define a set of `feature()` symbols used in the simulation.